### PR TITLE
Vaccine coverage last bits

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard8**
## Summary

- Fix the schema / validation error
- Add an inline tooltip for 60-64
- Fix date in footer

### CODE REVIEW

File: packages/app/src/pages/landelijk/vaccinaties.tsx [hunk 1]
```diff
@@ -200,8 +200,8 @@ const VaccinationPage = (props: StaticProps<typeof getStaticProps>) => {
               }
               metadata={{
                 datumsText: text.datums,
-                date: vaccine_coverage_per_age_group.last_value
-                  ?.date_of_report_unix,
+                date: vaccine_coverage_per_age_group.values[0]
+                  .date_of_report_unix,
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/pages/landelijk/vaccinaties.tsx:204]: you're going from the last value to the first value?

**Review by VWSCoronaDashboard1 — APPROVED** [review]

File: packages/app/src/pages/landelijk/vaccinaties.tsx (hunk 1 above)
Comment (VWSCoronaDashboard8) [reply packages/app/src/pages/landelijk/vaccinaties.tsx:204]: There is only one value. This is not a time series

Comment (VWSCoronaDashboard10) [reply packages/app/src/pages/landelijk/vaccinaties.tsx:204]: Ah right, shouldn't it be called `value` instead?

Comment (VWSCoronaDashboard1) [reply packages/app/src/pages/landelijk/vaccinaties.tsx:204]: I'm guessing the array has a length of one. But yea, probably safer to use last().

Comment (VWSCoronaDashboard8) [reply packages/app/src/pages/landelijk/vaccinaties.tsx:204]: Values are used differently here from time series. The array contains a value for each age group so nothing to do with time. I'm taking the timestamp from the first age group. All have the same timestamps, so it's completely redundant in how it was set up.

Comment (VWSCoronaDashboard10) [reply packages/app/src/pages/landelijk/vaccinaties.tsx:204]: Aaah of course. 🤦

**Pull request merged**